### ISSUE TITLE
Could not find a version that satisfies the requirement torch==0.4.1

### ISSUE THREAD

**Issue by 123Anu123** [dr-costas/mad-twinnet#13]
Downloaded the source code, and the pre-trained weights. I was running the requirements.txt file but an error keeps occuring when it reaches installation of pytorch version 0.4.1. Tried using conda and similar error occurs. Hence I can't move forward to use_me.py.

**Comment by Js-Mim** [comment]
There are two possible options:
1) Port it to a later version of pytorch. To the best of my knowledge, only the initialization methods have to be changed and the logging of the loss output.
2) Try installing it using `conda install pytorch=0.4.1 cudaVV -c pytorch` . Please replace "`VV`" with the appropriate CUDA version that your system is using. More info can be found in: https://pytorch.org/get-started/previous-versions/. 
Hope it helps!
